src/toplevel.pl

```diff
@@ -90,7 +90,7 @@
     delegate_task(Args, [Gs|Goals]).
 
 init_file :-
```

Comment:
I meant this `init_file/0` predicate: I think this should better be called `disable_init_file/0` instead of `init_file/0`, because the point is to disable the init&nbsp;file. 

`disabled_init_file/0`, i.e., disblade-**d** (with `d`), makes perfect sense for the fact, because then it has already be disabled. I would recommend to use an imperative name for predicates that do have side effects, like `disable_init_file/0` which asserts a fact. A non-imperative name is better for predicates that do not have side-effects, like `disabled_init_file/0`.

Reply:
Sorry, I fixed it now. I got confused yesterday